tables/sv-g0.utb

```diff
@@ -0,0 +1,515 @@
+# liblouis: Swedish Grade Uncontracted Braille Table
+#
+#-name: Svensk 6-punkts fullskrift
+#-index-name: Swedish, uncontracted
+#-display-name: Swedish uncontracted braille
+#
+#  Copyright (C) 2022 Insyn Scandinavia AB <david@insyn.se>
+#  Copyright (C) 2012 Mesar Hameed <mesar.hameed@gmail.com>
+#  Copyright (C) 2004-2008 ViewPlus Technologies, Inc. www.viewplus.com
+#  Copyright (C) 2004-2006 JJB Software, Inc. www.jjb-software.com
+#
+#-copyright: 2004-2006, JJB Software, Inc. <www.jjb-software.com>
+#-copyright: 2004-2008, ViewPlus Technologies, Inc. <www.viewplus.com>
+#-copyright: 2012, Mesar Hameed <mesar.hameed@gmail.com>
+#-copyright: 2022, Insyn Scandinavia AB <david@insyn.se>
+#
+#  This braille table is a heavily updated version of the former
+#  Swedish braille table called Se-Se-G1.utb which was quite outdated and
+#  was missing a lot of character definitions as well as incorrectly
+#  handling some situations regarding mixing of numbers and letters
+#  etc. This version also includes support for Swedish contractions
+#  which previously haven't been available in Liblouis.
+#  This new braille table is the result of a joint effort between
+#  Punktskriftsnämnden (the Swedish Braille Authority) and
+#  Insyn Scandinavia AB.
+#
+#-author-name: Insyn Scandinavia AB
+#-maintainer-name: Insyn Scandinavia AB
+#-maintainer-email: david@insyn.se
+#
+#  This braille table is based on the following documentation:
+#  Svenska skrivregler för punktskrift:
+#    https://www.mtm.se/skrivregler/dtb-bm-l1.html
+#  Kortskrift nivå 1 och 2:
+#    https://www.mtm.se/contentassets/08089b067f61495da1b035390d7a120e/kortskrift-niva-1-och-2---1997---nytt-manuskript-2016-09-26_v4.pdf
+#  
+#  This file is part of liblouis.
+#
+#  liblouis is free software: you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as
+#  published by the Free Software Foundation, either version 2.1 of the
+#  License, or (at your option) any later version.
+#
+#  liblouis is distributed in the hope that it will be useful, but
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with liblouis. If not, see
+#  <http://www.gnu.org/licenses/>.
+#
+#-license: LGPLv2.1
+
+#+language: sv
+#+type: literary
+#+contraction: no
+#+grade: 0
+#+dots: 6
+#+direction: both
```

Comment:
Oh, @InsynScandinavia one more thing: the table is marked as suited for both forward and backward translation, but there are only tests for forward translation.

Reply:
@InsynScandinavia Just making sure if you saw this question. If we could try wrapping up this PR before tomorrow that would be nice because I'm leaving on vacation. Christian will do the release on September 5 but we'd like to have everything ready well in time.